README.md

```diff
@@ -107,6 +107,17 @@ Install without RVM
 katello-devel-installer --katello-use-rvm false
 ```
 
+### From Git
+
+If you're working off the master branch, use Librarian to pull in all the modules to your local checkout:
```

Comment:
Could we include installation of librarian-puppet and puppet ruby > 2.0 since that isn't obvious?